### ISSUE TITLE
Model.update() does not update 'updateAt' attribute.

### ISSUE THREAD

**Issue by aloof-ruf** [automategreen/dynamoose#84]
I noticed that calling `SomeModel.update(...)` does not actually update the `updateAt` attribute created when you specify `timestamps: true` in a schema. For example:

**apple.js** (define our schema)
```
var dynamoose = require('dynamoose');

dynamoose.AWS.config.update({
  region: 'whatever-you-have'
});

dynamoose.local();

var appleSchema = new dynamoose.Schema({
  id: {
    hashKey: true,
    type: Number,
    required: true
  },
  appleType: {
    type: String,
    default: 'spartan',
    required: true,
    trim: true
  }
},
{
  timestamps: true
});

var apple = dynamoose.model('Apple', appleSchema);

module.exports = { 
   apple: apple
};
```

**index.js** (perform database operations)
```
var appleModel = require('./apple.js').apple;

console.log("Apples Test");

function updateApple1 () {
  appleModel.update({id: 1}, {appleType: 'gala'}, function (error, result) {
    if (error) return console.error(error);
    console.dir(result);
  });
}

appleModel.batchPut([
  {id: 1, appleType: 'granny smith'},
  {id: 2, appleType: 'spartan'},
  {id: 3, appleType: 'red delicious'}
], function (error, apples) {
  if (error) return console.error(error);
  console.dir(apples);

  setTimeout(updateApple1, 5000);
});
```

Now, when I call `aws dynamodb scan --table-name Apple --endpoint-url http://localhost:8000` I receive:
```
{
    "Count": 3, 
    "Items": [
        {
            "appleType": {
                "S": "spartan"
            }, 
            "id": {
                "N": "2"
            }, 
            "createdAt": {
                "N": "1481234294220"
            }, 
            "updatedAt": {
                "N": "1481234294220"
            }
        }, 
        {
            "appleType": {
                "S": "gala"
            }, 
            "id": {
                "N": "1"
            }, 
            "createdAt": {
                "N": "1481234294219"
            }, 
            "updatedAt": {
                "N": "1481234294220"
            }
        }, 
        {
            "appleType": {
                "S": "red delicious"
            }, 
            "id": {
                "N": "3"
            }, 
            "createdAt": {
                "N": "1481234294220"
            }, 
            "updatedAt": {
                "N": "1481234294220"
            }
        }
    ], 
    "ScannedCount": 3, 
    "ConsumedCapacity": null
}
```

Notice how the `updatedAt` attribute for the apple with id: 1 has not changed (still about the same as the `createdAt` attribute) despite the fact that I updated the item roughly five seconds after I created it.

This is contrary to what is expected from that attribute when you define `{timestamps: true}` in a schema.

**Comment by brandongoode** [comment]
Same root issue as #85.  We'll track it with that issue. Thanks.

**Comment by brandongoode** [comment]
Reopening -- #85 will require some additional options that are different than just making sure updateAt is updated.

**Comment by brandongoode** [comment]
Closed by #88
